fluentd/install.sls

```diff
@@ -53,7 +58,10 @@ fluentd_control_script:
 configure_fluentd_service:
   file.managed:
     - name: {{ fluentd_service.destination_path }}
-    - source: salt://fluentd/files/{{ fluentd_service.source_path }}
+    - source: salt://fluentd/templates/{{ fluentd_service.source_path }}
+    - context:
+        User: {{ fluentd.user }}
+        Group: {{ fluentd. group }}
```

Comment:
The keys in this context dict need to be lower cased match the case used in the template.

Reply:
I was mistakenly matching them against the real keys in the template file. Will fix shortly.